packages/block-editor/src/hooks/align.native.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import { without } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { addFilter } from '@wordpress/hooks';
+import { hasBlockSupport } from '@wordpress/blocks';
+
+const SUPPORTED_WIDE_ALIGNMENTS_BLOCKS = [
+	'core/cover',
+	'core/group',
+	'core/image',
+];
+const WIDE_ALIGNMENTS = [ 'wide', 'full' ];
```

Comment:
Can we store it in some kind of util to have access to it from other files as well? 
I've noticed there are some places where you compare data to ` === 'wide'` or ` === 'full'`. Also maybe it's worth to refactor it `ALIGNMENTS` object or something similar 🤔 :

```
const ALIGNMENTS = {
	widthAlignments: {
		wide: 'wide',
		full: 'full'
	},
	supportedWideBlocks: [
		'core/cover',
		'core/group',
		'core/image'
	]
	...
}
```

Reply:
Good idea! Thanks for the suggestions Luke! I'll work on the changes 👍